api/analysis/safety.go

```diff
@@ -78,42 +108,16 @@ func SafetyStartAnalysis(CID string, cOutput string) {
 		return
 	}
 
-	// step 2: finds Vulnerabilities
-	cResult = "passed"
-	if len(safetyOutput.SafetyIssues) != 0 {
-		cResult = "failed"
-	}
-
-	// step 3: update analysis' cResult into AnalyisCollection.
+	// issues found. client will have to handle with warnings and issues.
 	updateContainerAnalysisQuery := bson.M{
 		"$set": bson.M{
-			"containers.$.cResult": cResult,
+			"containers.$.cResult": "failed",
 		},
 	}
 	err = UpdateOneDBAnalysisContainer(analysisQuery, updateContainerAnalysisQuery)
 	if err != nil {
 		log.Error("SafetyStartAnalysis", "SAFETY", 2007, err)
-		return
-	}
-}
-
-// StringToLastLine receives a string with multiple lines and returns it's last
-func StringToLastLine(s string) string {
-	var lines []string
-	scanner := bufio.NewScanner(strings.NewReader(s))
-	for scanner.Scan() {
-		lines = append(lines, scanner.Text())
 	}
-	return lines[len(lines)-1]
-}
+	return
 
-// GetAllLinesButLast receives a string with multiple lines and returns all but the last line.
-func GetAllLinesButLast(s string) []string {
```

Comment:
These utils functions will be place into `util` package as both client and api uses them.